docs/how-to-release.md

```diff
@@ -7,11 +7,13 @@
     * Run `python scripts/ci/update_release.py` to update the relevant yaml manifests.
     * Copy `CRD`s to Helm Chart
       - `cp config/crd/bases/mongodbcommunity.mongodb.com_mongodbcommunity.yaml helm-charts/charts/community-operator-crds/templates/mongodbcommunity.mongodb.com_mongodbcommunity.yaml`
-    * Commit all changes.
-    * Create a PR with the title `Release MongoDB Kubernetes Operator v<operator-version>` (the title must match this pattern)
-    
-* Have this PR Reviewed and Approved.
-    * Upon approval, all new images for this release will be built and released, and a Github release draft will be created.
-    * Once tests have passed, merge the release PR.
-
-* Review and publish the new GitHub release draft.
+      - commit changes to the [helm-charts submodule](https://github.com/mongodb/helm-charts) and create a PR against it ([similar to this one](https://github.com/mongodb/helm-charts/pull/163)).
+      - do not merge helm-charts PR until release PR is merged and the images are pushed to quay.io.
+    * Commit all changes
+      * This also includes helm-chart submodule update (to the commit pointing in the helm-chart PR)
+    * Create a PR with the title `Release MongoDB Kubernetes Operator v<operator-version>` (the title must match this pattern).
+    * Wait for the tests to pass and merge the PR.
+      * Upon approval, all new images for this release will be built and released, and a GitHub release draft will be created.
+      * Review and publish the new GitHub release draft, that was prepared
+    * Merge helm-charts PR and update submodule to the latest commit on `main` branch.
+    * Create a new PR with only bump to the helm-chart submodule.
```

Comment:
Ideally I would like to have helm-chart PR merged before merging the release PR. But this way, we're publishing new helm-chart that is referencing new quay.io images which don't exist yet. So this opens a window, that someone would not be able to install the operator from the newest image. Hence this two step process of merging and bumping.